src/html-media-source.js

```diff
@@ -79,6 +79,8 @@ export default class HtmlMediaSource extends videojs.EventTarget {
       // Retain the reference but empty the array
       this.activeSourceBuffers_.length = 0;
 
+      // By default, disable the audio in the combined virtual source buffer
+      // and enable the audio-only source buffer.
       let combined = true;
```

Comment:
`combined` should start out `false` and `audioOnly` should start out `true`, since they represent `audioDisabled_` and that is the default.